test/verify/check-system-info

```diff
@@ -911,6 +911,11 @@ password=foobar
         profile = m.execute(cmd + " --show").strip()
         b.wait_text("#crypto-policy-button", shown_profile_text(profile))
 
+        # Test if a new subpolicy can be set
+        new_profile = "DEFAULT:SHA1"
+        change_profile(profile, new_profile)
+
+        profile = m.execute(cmd + " --show").strip()
```

Comment:
Missing assertion of `profile`?

Reply:
This is still current. The test only validates that the UI reflects `--show`, but it should assert that --show *actually* says `new_profile` now.

Reply:
There is an assertion below right now?

Reply:
thanks!